src/wp-admin/widgets-form.php

```diff
@@ -211,7 +211,7 @@
 			$id_base      = implode( '-', $pieces );
 			$widget       = get_option( 'widget_' . $id_base );
 			unset( $widget[ $multi_number ] );
-			update_option( 'widget_' . $id_base, $widget );
+			update_option( 'widget_' . $id_base, $widget, false );
```

Comment:
This still needs to be addressed per https://github.com/WordPress/wordpress-develop/pull/6465#discussion_r1731918565.
```suggestion
			update_option( 'widget_' . $id_base, $widget );
```